### PR TITLE
Bumped to Akka Stream version 1.0-M3

### PR DESCRIPTION
**Pull request by drewhk**

### CODE REVIEW

**Comment by ktoso** [comment]
yup, LGTM

**Comment by patriknw** [comment]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -39,21 +39,21 @@ Automatically published Scaladoc API for the latest SNAPSHOT version of Akka can
...
-  * `"com.typesafe.akka" % "akka-stream-experimental_2.10" % "1.0-M2"`
+  * `"com.typesafe.akka" % "akka-http-experimental_2.10" % "1.0-M3"`
+  * `"com.typesafe.akka" % "akka-http-core-experimental_2.10" % "1.0-M3"`
+  * `"com.typesafe.akka" % "akka-stream-experimental_2.10" % "1.0-M3"`
 
-* API documentation for [Java](http://doc.akka.io/japi/akka-stream-and-http-experimental/1.0-M2/) and [Scala](http://doc.akka.io/api/akka-stream-and-http-experimental/1.0-M2/)
+* API documentation for [Java](http://doc.akka.io/japi/akka-stream-and-http-experimental/1.0-M3/) and [Scala](http://doc.akka.io/api/akka-stream-and-http-experimental/1.0-M3/)
 
-* Reference documentation [Scala](http://doc.akka.io/docs/akka-stream-and-http-experimental/1.0-M2/scala.html)
+* Reference documentation [Java](http://doc.akka.io/docs/akka-stream-and-http-experimental/1.0-M3/java.html) and [Scala](http://doc.akka.io/docs/akka-stream-and-http-experimental/1.0-M3/scala.html)
```
Comment (rkuhn) [inline docs/index.md]: there’s a `for` missing here

**Comment by rkuhn** [comment]
LGTM, publish when amended

**Pull request merged**